apps/yapms/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -65,6 +66,36 @@ function findCandidate(id: string | null) {
 	}
 }
 
+function getCandidatesForRegion(candidateStr: string | null, value: number) {
+	if (candidateStr === null) {
+		return [{ candidate: get(TossupCandidateStore), count: value, margin: 0 }];
+	}
```

Comment:
Technically it should never hit this case- I put this here to avoid a bit of a messy conditional, but this code works as well and is nicer, will revise:

```
const candidateString = childHTML.getAttribute('candidates');
candidates: candidateString !== null //God bless our linting overlords.
	? getCandidatesForRegion(candidateString, value)
	: [{ candidate: tossupCandidate, count: value, margin: 0 }],
```